test/widgets/login_page_test.dart

```diff
@@ -0,0 +1,235 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/util/buttons/loading_button.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:motis_mitfahr_app/welcome/pages/forgot_password_page.dart';
+import 'package:motis_mitfahr_app/welcome/pages/login_page.dart';
+import 'package:motis_mitfahr_app/welcome/pages/register_page.dart';
+import 'package:progress_state_button/progress_button.dart';
+
+import '../util/mocks/mock_server.dart';
+import '../util/mocks/request_processor.dart';
+import '../util/mocks/request_processor.mocks.dart';
+import '../util/pump_material.dart';
+
+void main() {
+  final MockRequestProcessor processor = MockRequestProcessor();
+  const String email = 'motismitfahrapp@gmail.com';
+  const String password = 'abc&EFG0';
+
+  setUpAll(() async {
+    MockServer.setProcessor(processor);
+    SupabaseManager.setCurrentProfile(null);
+  });
+
+  group('LoginPage', () {
+    final Finder emailFieldFinder = find.descendant(
+      of: find.byKey(const Key('loginEmailField')),
+      matching: find.byType(TextFormField),
+    );
+    final Finder passwordFieldFinder = find.descendant(
+      of: find.byKey(const Key('loginPasswordField')),
+      matching: find.byType(TextFormField),
+    );
+    final Finder loginButtonFinder = find.byType(LoadingButton);
+
+    testWidgets('Shows the login page', (WidgetTester tester) async {
```

Comment:
```suggestion
    testWidgets('Shows the login form', (WidgetTester tester) async {
```

Reply:
I mean, yeah sure but it also shows the page